### PR TITLE
Adding AKKA HTTP Training

### PR DESCRIPTION
**Pull request by taljmars**

### CODE REVIEW

**Comment by taljmars** [comment]
@maryum375  Can you please review ?

File: scala2-akka-untitled/src/main/scala/AkkaHttp/Actors/FIleSystemActor.scala [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import akka.actor.Actor
+import akka.stream.IOResult
+import akka.stream.scaladsl.{FileIO, Source}
+import akka.util.ByteString
+
+import java.nio.file.Paths
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class FIleSystemActor extends Actor {
```
Comment (maryum375) [inline scala2-akka-untitled/src/main/scala/AkkaHttp/Actors/FIleSystemActor.scala:13]:
```suggestion
class FileSystemActor extends Actor {
```
:)

File: scala2-akka-untitled/src/main/scala/AkkaHttp/Actors/ImageDownloadActor.scala [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  implicit val system = context.system
+
+  override def receive: Receive = {
+
+    case DOWNLOAD(url) =>
+        val originalSender = sender()
+        Http(context.system).singleRequest(Get(url)).map {
+          case HttpResponse(StatusCodes.OK, _, entity, _) =>
+            val imageStringFuture: Future[String] = entity.dataBytes.runFold[ByteString](ByteString(""))(_ ++ _).map {
```
Comment (maryum375) [inline scala2-akka-untitled/src/main/scala/AkkaHttp/Actors/ImageDownloadActor.scala:28]: Don't like this type of writing `_ ++ _`
Although it's vert scallaish 🤣

**Review by maryum375 — APPROVED** [review]
Very cool project. Can be even used for. future reference in the onboarding process for a simple actor based project.

Good job!

**Pull request merged**